docs/volume-condition.md

```diff
@@ -0,0 +1,100 @@
+# Volume Condition Reporter
```

Comment:
Maybe we can link to these docs from the main README.md

https://github.com/csi-addons/kubernetes-csi-addons?tab=readme-ov-file#csi-driver-side-car

Reply:
done, see PR #842